lib/AFS/CellCC/Dump.pm

```diff
@@ -354,6 +374,79 @@ process_dumps($$$@) {
         # our caller.
         exit(1);
     }
+    $pm->wait_all_children();
```

Comment:
this isn't quite what I meant when I said before we could do the _calc_incremental call, and then wait for the children to finish. By "waiting for them to finish" I meant, we should examine the state of the jobs to see if they are done running, and start running dumps when they're done. By waiting for the actual child procs to finish, we stall the whole dump-server until they're all done; so if something gets stuck for any reason (e.g. a `vos` command hanging), we don't start any new dumps. That's not good.

Instead what I suggest here is having the `_calc_incremental` calls be done in a new state, and then we can check to see if any such jobs are still running in that state for that volume.